cypress/integration/console/gateways/create.spec.js

```diff
@@ -0,0 +1,98 @@
+
+  before(() => {
+    cy.dropAndSeedDatabase()
+    cy.createUser(user)
+  })
+
+  beforeEach(() => {
+    cy.loginConsole({ user_id: user.ids.user_id, password: user.password })
+  })
+
+  it('displays UI elements in place', () => {
+    cy.visit(`${Cypress.config('consoleRootPath')}/gateways/add`)
+
+    cy.findByText('Add gateway', { selector: 'h1' }).should('be.visible')
+    cy.findByLabelText('Gateway ID').should('be.visible')
+    cy.findByLabelText('Gateway EUI').should('be.visible')
+    cy.findByLabelText('Gateway name').should('be.visible')
+    cy.findByLabelText('Gateway description').should('be.visible')
+    cy.findDescriptionByLabelText('Gateway description')
+      .should(
+        'contain',
+        'Optional gateway description; can also be used to save notes about the gateway',
+      )
+      .and('be.visible')
+    cy.findByLabelText('Gateway Server address').should('be.visible')
+    cy.findDescriptionByLabelText('Gateway Server address')
+      .should('contain', 'The address of the Gateway Server to connect to')
+      .and('be.visible')
+    cy.findByLabelText('Gateway status').should('be.visible')
+    cy.findDescriptionByLabelText('Gateway status')
+      .should('contain', 'The status of this gateway may be publicly displayed')
+      .and('be.visible')
+    cy.findByRole('button', { name: /Add attributes/ }).should('be.visible')
+    cy.findByLabelText('Frequency plan').should('be.visible')
+    cy.findByLabelText('Schedule downlink late').should('be.visible')
+    cy.findByLabelText('Duty cycle').should('be.visible')
+    cy.findByTestId('schedule_anytime_delay').should('be.visible')
+    cy.findByLabelText('Automatic updates').should('be.visible')
+    cy.findDescriptionByLabelText('Automatic updates')
+      .should('contain', 'Gateway can be updated automatically')
+      .and('be.visible')
+    cy.findByLabelText('Channel').should('be.visible')
+    cy.findByRole('button', { name: 'Create gateway' }).should('be.visible')
+  })
+
+  it('validates before submitting an empty form', () => {
+    cy.visit(`${Cypress.config('consoleRootPath')}/gateways/add`)
+
+    cy.findByRole('button', { name: 'Create gateway' })
+      .should('be.visible')
+      .click()
+
+    cy.findErrorByLabelText('Gateway ID')
+      .should('contain.text', 'Gateway ID is required')
+      .and('be.visible')
+    cy.location('pathname').should('eq', `${Cypress.config('consoleRootPath')}/gateways/add`)
+  })
+
+  it('successfully adds gateway', () => {
+    cy.visit(`${Cypress.config('consoleRootPath')}/gateways/add`)
+    cy.findByLabelText('Gateway ID').type(gatewayId)
+    cy.findByLabelText('Frequency plan').selectOption(frequencyPlanId)
+
+    cy.findByRole('button', { name: 'Create gateway' })
+      .should('be.visible')
+      .click()
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.location('pathname').should(
+      'eq',
+      `${Cypress.config('consoleRootPath')}/gateways/${gatewayId}`,
+    )
+  })
```

Comment:
```suggestion
  cy.findByTestId('full-error-view').should('not.exist')
  })
```

This is important to check too. I saw it slipped through the cracks for application and organization tests as well. Please add them there as well.